src/components/index.js

```diff
@@ -5,65 +5,65 @@
  */
 export Anontools from '@plone/volto/components/theme/Anontools/Anontools';
 export Breadcrumbs from '@plone/volto/components/theme/Breadcrumbs/Breadcrumbs';
-export ContactForm from '@plone/volto/components/theme/ContactForm/ContactForm';
+// export ContactForm from '@plone/volto/components/theme/ContactForm/ContactForm';
```

Comment:
@pnicolli @sneridagh @robgietema this is technically a breaking change, correct? Do we have to assume ppl rely on those imports already? This has nothing to do with this PR really. We just have to somehow define our Volto API for semantic versioning/releases and upgrade guides.

Reply:
@tisto It is, but for code splitting work as expected, we need to remove them which kind of sucks... Unless we find a better/elegant way.

Reply:
Yeah, that's a breaking change :( Also, my current implementation is absolutely non-elegant, it would be awesome to find a better one. One components index per split bundle, for example, but that would mean having a better idea of what bundles we will split.